### PR TITLE
update COS link for Dockerfile and README

### PR DESCRIPTION
**Pull request by I-Hong**
Update links in Dockerfile and README

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,10 +14,10 @@
 # limitations under the License.
 #
 
-FROM codait/max-base:v1.1.1
+FROM codait/max-base:v1.1.3
 
 # Fill in these with a link to the bucket containing the model and the model file name
-ARG model_bucket=http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/facial-age-estimator/1.0
+ARG model_bucket=http://s3.us-south.cloud-object-storage.appdomain.cloud/max-assets-prod/max-facial-age-estimator/1.0.0
```
Comment (bdwyer2) [inline Dockerfile]:
```suggestion
ARG model_bucket=https://s3.us-south.cloud-object-storage.appdomain.cloud/max-assets-prod/max-facial-age-estimator/1.0.0
```

Comment (I-Hong) [reply Dockerfile]: Thanks @bdwyer2. Updated.

**Comment by I-Hong** [comment]
I've also updated COS. 
> Looks like you need to set the file to public on COS. Once that is done this looks good to me!

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
-The model is based on the [SSR-Net model](https://github.com/shamangary/SSR-Net). The model files are hosted on [IBM Cloud Object Storage](http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/facial-age-estimator/1.0/assets.tar.gz). The code in this repository deploys the model as a web service in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
+The model is based on the [SSR-Net model](https://github.com/shamangary/SSR-Net). The model files are hosted on [IBM Cloud Object Storage](http://s3.us-south.cloud-object-storage.appdomain.cloud/assets-prod/max-facial-age-estimator/1.0.0/assets.tar.gz). The code in this repository deploys the model as a web service in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
```
Comment (bdwyer2) [inline README.md]:
```suggestion
The model is based on the [SSR-Net model](https://github.com/shamangary/SSR-Net). The model files are hosted on [IBM Cloud Object Storage](http://s3.us-south.cloud-object-storage.appdomain.cloud/max-assets-prod/max-facial-age-estimator/1.0.0/assets.tar.gz). The code in this repository deploys the model as a web service in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
```

**Comment by splovyt** [comment]
Can you bump the version number as well please?

**Comment by I-Hong** [comment]
Updated.

**Pull request merged**